build.assets/Dockerfile

```diff
@@ -107,10 +110,12 @@ RUN apt-get update -y --fix-missing && \
     rm -rf /var/lib/apt/lists/*
 
 # Install gcloud SDK and Firestore emulator.
-RUN curl -sSL https://sdk.cloud.google.com | bash -s -- --install-dir=/opt --disable-prompts
+RUN (curl -sSL https://sdk.cloud.google.com | bash -s -- --install-dir=/opt --disable-prompts) && \
+    rm -rf /opt/google-cloud-sdk/.install/.backup
 ENV PATH="$PATH:/opt/google-cloud-sdk/bin"
 # beta component is required by firestore emulator: https://cloud.google.com/sdk/gcloud/reference/beta/emulators/firestore/start
-RUN gcloud components install cloud-firestore-emulator beta
+RUN gcloud components install cloud-firestore-emulator beta && \
```

Comment:
Maybe set the PATH up front and condense these two steps into one, so we only remove the cache once?

Reply:
Done.